Track Touch Events in web-sys Backend (#1900)
rust-windowing/winit: CHANGELOG.md

# Unreleased

> - Add support for touch events in the web-sys backend.

Reviewer: ```suggestion
- On Web (web-sys only), add support for touch events.
```
